packages/nimbus/package.json

```diff
@@ -54,26 +54,22 @@
     "@pandacss/types": "catalog:react",
     "@react-aria/optimize-locales-plugin": "catalog:react",
     "@storybook/addon-a11y": "catalog:tooling",
-    "@storybook/addon-essentials": "catalog:tooling",
-    "@storybook/experimental-addon-test": "catalog:tooling",
-    "@storybook/preview-api": "catalog:tooling",
-    "@storybook/react": "catalog:tooling",
+    "@storybook/addon-vitest": "catalog:tooling",
     "@storybook/react-vite": "catalog:tooling",
-    "@storybook/test": "catalog:tooling",
     "@testing-library/react": "catalog:tooling",
     "@testing-library/user-event": "catalog:tooling",
     "@types/react": "catalog:react",
     "@types/react-dom": "catalog:react",
     "@vitejs/plugin-react": "catalog:tooling",
     "@vitest/browser": "catalog:tooling",
     "@vitest/coverage-v8": "catalog:tooling",
+    "@vueless/storybook-dark-mode": "catalog:tooling",
```

Comment:
`storybook-dark-mode` seems to be [abandoned](https://github.com/hipstersmoothie/storybook-dark-mode/issues/295#issuecomment-2938151892).

`@vueless/storybook-dark-mode` is a maintained fork that is [updated for storybook 9](https://github.com/vuelessjs/storybook-dark-mode?tab=readme-ov-file#storybook-dark-mode)